std/conv.d

```diff
@@ -3737,10 +3737,21 @@ unittest
 @property T octal(T, string num)()
     if (isOctalLiteral!num)
 {
+    return octal!T(num);
+}
+
+/// Ditto
+@property T octal(T)(const(char)[] num) pure nothrow
```

Comment:
Can we make this a separate pull? I don't think all the kinks of using generic char ranges are worked out yet, or am I wrong?